hack/api-reference/config.md

```diff
@@ -57,9 +57,7 @@ string
 <td>
 <code>clientConnection</code></br>
 <em>
-<a href="https://godoc.org/k8s.io/component-base/config/v1alpha1#ClientConnectionConfiguration">
-Kubernetes v1alpha1.ClientConnectionConfiguration
-</a>
+invalid type
```

Comment:
why do we have here `invalid type`?

Reply:
This file is auto-generated. I regenerated using `make generate` and don't see `invalid type` anymore. I already asked to run `make generate`.

Reply:
I see